connectors/tests/test_basic_rule.py

```diff
@@ -1187,3 +1194,52 @@ def test_is_not_include_for_exclude_policy():
     )
 
     assert not basic_rule.is_include()
+
+
+def basic_rule_json(merge_with=None, delete_keys=None):
```

Comment:
No need to review (temporary copy from https://github.com/elastic/connectors-python/pull/268 until it's merged)